### PR TITLE
WD-12827 - rebrand /azure/private-offer

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Rebrand /azure/private-offer. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14156.demos.haus/azure/private-offer)
- [figma](https://www.figma.com/design/MinLlXouBnEDqaoAL3bTWC/24.10-Azure-bubble?node-id=1-6057&t=gyQr0wjIOwcXzRPS-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12827](https://warthogs.atlassian.net/browse/WD-12827)

[WD-12827]: https://warthogs.atlassian.net/browse/WD-12827?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mattea-turic** [comment]
@lizzochek I added a couple minor comments in the figma file

**Comment by mattea-turic** [comment]
@lizzochek padding is good now! but the images still don't stretch over 6 columns (showing as 564x376 rather than 600x400), but lmk if this is to do with the asset as I can export one that has the right dimensions

File: templates/azure/shared/_how-ubuntu-pro-helps-you.html [hunk 1]
```diff
@@ -1,82 +1,25 @@
...
-    }}
-
-    <p>10 years of extended maintenance</p>
-  </div>
+<div class="col">
+  <h2>How Ubuntu Pro helps you</h2>
+</div>
+<div class="col">
+  <ul class="p-list--divided u-no-margin--bottom">
+    <li class="p-list__item is-ticked" style="box-shadow: unset;">
```
Comment (mtruj013) [inline templates/azure/shared/_how-ubuntu-pro-helps-you.html:6]: I thought this was for the hr above the first divided list item bug that's fixed now in prod, but actually it looks like this isn't doing anything. Either way looks safe to remove!
```suggestion
    <li class="p-list__item is-ticked">
```

File: templates/azure/shared/_how-ubuntu-pro-helps-you.html [hunk 2]
```diff
@@ -1,82 +1,25 @@
...
+      <p class="u-no-padding u-no-margin--bottom">
```
Comment (mtruj013) [inline templates/azure/shared/_how-ubuntu-pro-helps-you.html:7]: Why wrap the content in an additional `p` without any of it's original spacing? Seems like the same thing could be accomplished by not using it at all

**Review by mtruj013 — COMMENTED** [review]
Thanks @lizzochek! Some suggestions/questions, nothing major though

File: templates/azure/shared/_how-ubuntu-pro-helps-you.html (hunk 2 above)
Comment (lizzochek) [reply templates/azure/shared/_how-ubuntu-pro-helps-you.html:7]: It still saves some margin there

File: templates/azure/private-offer.html [hunk 3]
```diff
@@ -1,132 +1,167 @@
...
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <hr />
+      <h2 class="p-text--small-caps">Get Ubuntu Pro for Azure</h2>
+      <div class="p-logo-section">
+        <div class="p-logo-section__items u-no-margin--bottom">
+          <div class="p-logo-section__item">
+            {{ image (
```
Comment (mtruj013) [inline templates/azure/private-offer.html:38]: For all other instances as well pls
```suggestion
            {{ image(
```

**Review by mtruj013 — APPROVED** [review]
Thanks @lizzochek! Looks like there are a couple images that still need formatting, but other than that LGTM

**Pull request merged**